extensions/wikidata/src/org/openrefine/wikidata/commands/PreviewWikibaseSchemaCommand.java

```diff
@@ -105,10 +105,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             Engine engine = getEngine(request, project);
             List<ItemUpdate> editBatch = schema.evaluate(project, engine, warningStore);
 
-            // Inspect the edits and generate warnings
-            EditInspector inspector = new EditInspector(warningStore, manifest);
-            inspector.inspect(editBatch);
-            
+            if (manifest.getConstraintsRelatedId("property_constraint_pid") != null) {
```

Comment:
Not all scrutinizers rely on WikibaseQualityConstraints, so it is worth inspecting the batch even if no constraints are set up in the manifest.

Reply:
Fixed with commit 4d9cc4e below.